tests/bud.bats

```diff
@@ -3818,6 +4014,20 @@ _EOF
   [[ "$output" -gt 1 ]] # should at least be more than one entry in there, right?
 }
 
+@test "bud-multiple-platform for --all-platform with additional-build-context" {
+  outputlist=localhost/testlist
+    mkdir -p ${TEST_SCRATCH_DIR}/bud/platform
+
+cat > ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile1 << _EOF
+FROM busybox
+_EOF
+
+  # Pulled images must be ubi since we configured --build-context busybox=docker://registry.access.redhat.com/ubi8-micro
+  run_buildah build $WITH_POLICY_JSON --jobs=1 --all-platforms --build-context busybox=docker://registry.access.redhat.com/ubi8-micro --manifest $outputlist -f ${TEST_SCRATCH_DIR}/bud/platform/Dockerfile1
```

Comment:
`--jobs=1` is the default, so it doesn't need to be specified.  This test should also compare how many entries end up in `$outputlist` to the number of images in the registry's `ubi8-micro` manifest list.

Reply:
Checking the length of manifest list now it should be > 1 just like we do in `no-run` test.

Reply:
`--jobs=1` is the default, so it doesn't need to be specified.